helper/helper.go

```diff
@@ -69,6 +69,36 @@ func EncryptMessageArmored(key, plaintext string) (ciphertext string, err error)
 	return ciphertext, nil
 }
 
+// EncryptRawMessageArmored generates an armored PGP message given a rawdata and
+// an armored public key.
+func EncryptRawMessageArmored(key string, data []byte) (string, error) {
```

Comment:
The function contains a lot of duplicated code with `EncryptMessageArmored`, could you create a common internal function that encrypts given the `PlainMessage` and use it in both?